unfetter-discover/unfetter-ui
# Issue 1260

Fixes unfetter-discover/unfetter#1260

Upon edit of an existing baseline, we'll skip to the first page of the wizard which has data to be entered.  This would be either the groups page or the scoring panel.  For the scoring panel, an `unfinished` page has one or more un-set PDR scores (i.e. empty).

### To test:
1. Pull this branch
2. Open an existing baseline
3. First page will be groups page if all scores are entered, or the appropriate scoring panel for a capability that has the first empty scoring value
4. Create a baseline with no groups or capabilities selected, save it, and edit it.  You should be left at the Capability Groups page.
5. Create a baseline with groups and capabilities chosen, and with varying empty scoring values.
6. As you fill in the scoring values, the first panel on baseline edit would change.
